src/ThreeEditor/components/Sidebar/properties/category/ParticleFilterConfiguration.tsx

```diff
@@ -0,0 +1,46 @@
+import { Object3D } from 'three';
```

Comment:
Do we need to have a link to 3D editor here ?

Reply:
We need if we are going to use smartwatcheditor

Reply:
OK